pkg/kubelet/util/manager/manager.go

```diff
@@ -46,16 +47,18 @@ type Manager interface {
 
 // Store is the interface for a object cache that
 // can be used by cacheBasedManager.
+// Since tenant-namespae-name trinity can't uniquely identify a resource in scale-out env,
+// one more factor, origin ID, is needed, representing the origin API server identifier.
 type Store interface {
 	// AddReference adds a reference to the object to the store.
 	// Note that multiple additions to the store has to be allowed
 	// in the implementations and effectively treated as refcounted.
-	AddReference(tenant, namespace, name string)
+	AddReference(tenant, namespace, name string, originId int)
```

Comment:
originId is a very generic name. Consider use straight forward name tenantPartitionId

Reply:
yes will rename in the consistent way